### PR TITLE
Logout functionality and welcome_email_sent

### PR DESCRIPTION
**Pull request by biancaflaidar**

### CODE REVIEW

File: app/scripts/controllers/signup.js [hunk 1]
```diff
@@ -161,6 +161,8 @@ angular.module('adminAppFrontendApp')
                         }
 
                         var hotelID = 'private_' + authData.uid;
+                        
+                        newProperty.welcome_email_sent = false;
```
Comment (shakeelmohamed) [inline app/scripts/controllers/signup.js]: This should be on the user, not the property.

**Comment by shakeelmohamed** [comment]
Logout stuff looks good, the welcome email thing should be set on the user, not the property

**Comment by biancaflaidar** [comment]
Updated

File: app/scripts/controllers/signup.js [hunk 2]
```diff
@@ -131,12 +131,13 @@ angular.module('adminAppFrontendApp')
...
                 delete newUser.password;
                 delete $scope.newUser.password;
-
+                
                 return ret;
             }).then(function(authData) {
                 console.log("Logged in as:", authData.uid);
 
                 // Create user data, mapped to the firebase generated ID
+                $scope.newUser.welcome_email_sent = false;
```
Comment (shakeelmohamed) [inline app/scripts/controllers/signup.js]: change this to `newUser.welcome_email_sent = false;`

**Comment by biancaflaidar** [comment]
Updated

**Comment by shakeelmohamed** [comment]
thanks

**Pull request merged**